.asf.yaml

```diff
@@ -0,0 +1,41 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+github:
+  description: SeaTunnel is a distributed, high-performance data integration platform for the synchronization and transformation of massive data (offline & real-time).
+  homepage: https://seatunnel.apache.org/
+  labels:
+    - data-integration
+    - high-performance
+    - offline&real-time
```

Comment:
I prefer to separate this into two words, `offline` `real-time`

Reply:
How about add one more label `data pipeline`

Reply:
Depending on how you understand it, I prefer one. Because we provide it at the same time.

Reply:
> How about add one more label `data pipeline`

We can add it later. If there are no other problems, I hope this can be merged as soon as possible. Dev mail has received too many commit messages.

Reply:
> Depending on how you understand it, I prefer one. Because we provide it at the same time.

IMO, labels have two functions, tell user/dev what is our focus on, the other is for search engines. And I prefer to separate them is for the search engine side, cause one or two words is both clearly for user/dev.

I am not sure we could hint in github/google in use one word, with the search `real time data integration` or some thing like that